superset-frontend/src/dashboard/containers/Chart.jsx

```diff
@@ -83,6 +83,8 @@ function mapStateToProps(
     formData,
     editMode: dashboardState.editMode,
     isExpanded: !!dashboardState.expandedSlices[id],
+    showFilterBadge: dashboardInfo.metadata?.showFilterBadge ?? true,
```

Comment:
may be it can good approach to switch it to `hideFilterBadge` instead and it will be used as `undefined` as `false` @villebro  thoughts

Reply:
I think it's usually better to avoid having negatives in option names. Let's just make sure we initialize with defaults and then make sure we do strict equality to avoid problems from `undefined` values.